Formula/passenger.rb

```diff
@@ -18,10 +18,24 @@ class Passenger < Formula
   depends_on "openssl"
   depends_on :macos => :lion
 
+  # macOS Sierra ships the APR libraries & headers, but has removed the
+  # apr-1-config & apu-1-config executables which are used to find
+  # those elements. We may need to adopt a broader solution if this problem
+  # expands, but currently subversion & passenger are the only breakage as a result.
+  if MacOS.version >= :sierra && (MacOS::Xcode.version.to_f >= 8.0 || MacOS::CLT.version.to_f >= 8.0)
```

Comment:
Just `if MacOS.version >= :sierra`. We don't support any configuration on Sierra that doesn't involve using Xcode 8 and/or CLT 8.x.